api/Cargo.toml

```diff
@@ -29,4 +29,4 @@ base64 = "0.13.0"
 rand = "0.8.5"
 
 shuttle-common = { version = "0.2.6", path = "../common" }
-shuttle-service = { version = "0.2.6", path = "../service", features = [ "loader" ] }
+shuttle-service = { version = "0.2.6", path = "../service", features = [ "loader", "sqlx-postgres" ] }
```

Comment:
Is it possible to not turn the postgres feature on? Maybe the secret store should be in common instead?

Reply:
But then that would surely just move the Postgres dependency into common instead - would that still be advantageous?